include/glow/Graph/Graph.h

```diff
@@ -46,6 +46,14 @@ using PlaceholderList = std::list<Placeholder *>;
 using UnsignedArrayRef = llvm::ArrayRef<size_t>;
 /// Map from original Nodes to cloned Nodes.
 using NodeMap = llvm::DenseMap<Node *, Node *>;
+/// State of a function. This can be used to control optimizations which depend
+/// on the state of the Function.
```

Comment:
Can you add a disclaimer comment making it very clear this is a temporary workaround until GH Issue #3213 is complete? I don't want people to start relying on this if we will remove it in the future.